src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -276,8 +259,10 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
   const crosshairEdgeMax = width * 0.85
   const crosshairAtEdge = !!crosshair && crosshair > crosshairEdgeMax
 
-  /* Default curve doesn't look good for the HOUR/ALL chart */
-  const curveTension = timePeriod === TimePeriod.ALL ? 0.75 : timePeriod === TimePeriod.HOUR ? 1 : 0.9
+  /* Default curve doesn't look good for the HOUR chart. 
```

Comment:
Reformat (I'm surprised your linter does not do this):

```
/*
 * Default curve doesn't look good for the HOUR chart.
 * Higher values make the curve more rigid, lower values smooth the curve but make it less "sticky" to real data points,
 * making it unacceptable for longer durations / larger variances.
 */
```

Reply:
oh weird need to fix my linter then. 

thanks done.